### ISSUE TITLE
PEP 541 Request: ao

### ISSUE THREAD

**Issue by esdandreu** [pypa/pypi-support#2247]
### Project to be claimed

`ao`: https://pypi.org/project/ao

### Your PyPI username

`esdandreu`: https://pypi.org/user/esdandreu

### Reasons for the request

The current project not only has been abandoned but it never was used. Only one version in the history, no description, no homepage. According to Google BigQuery this package has been downloaded 71 times in the last 30 days, only 21 of them through `pip`. According to [libraries.io](https://libraries.io/pypi/ao) no other package depends on this one. Taking into account the author's [other published packages](https://pypi.org/user/ooo/), I believe that this is a test package with no purpose.

In contrast, I intend to use the package name for an [Acoustic Odometry library](https://github.com/AcousticOdometry/AO). In the field of robot localization it is a standard to name the methods with an acronym of the input modalities: Visual Odometry = VO; Visual Inertial Odometry = VIO. Then, derived methods and variations tend to compose that acronym with another name: [Kimera-VIO](https://github.com/MIT-SPARK/Kimera-VIO). My project is a framework upon which almost any Acoustic Odometry method can be built. Therefore I would like to name it Acoustic Odometry = AO. The project is hosted in a [public organization](https://github.com/AcousticOdometry), and I intend to open it to other maintainers if it gets that kind of attention.

To sum up, I request to replace an unmaintained and empty project with a project built for the robot localization community.

Thank you for your time.

### Maintenance or replacement?

Replacement

### Source code repositories URLs

The current project has no source code repository.
My replacement is here [https://github.com/AcousticOdometry/AO](https://github.com/AcousticOdometry/AO)

### Contact and additional research

I tracked down the author of the package `ooo` using the github page of other packages under his name to the github account [`daiooo`](https://github.com/daiooo). Unfortunately, I couldn't find any contact information there and the account has no activity since 2020.

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

**Comment by yeraydiazdiaz** [comment]
Hi @esdandreu, I can confirm the project is part of a mass squat from the user as such I've transferred it to you without attempts of contact.
